build/mergeallmodules.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+#Assumes that ./x64 and ./x86 exist and contain the arch-specific copies of a DSC resource module
+
+set -e
+
+zip -L >/dev/null 2>&1
+if [ $? != 0 ]; then
+        echo "zip must be installed"
+        exit 1
+fi
+
+unzip -Z >/dev/null 2>&1
+if [ $? != 0 ]; then
+        echo "unzip must be installed"
+        exit 1
+fi
+
+TargetDir="./target"
+x64dir="./x64"
+x86dir="./x86"
```

Comment:
If this is intended to run from the build directory, then you should probably add:

```
/build/target
/build/x64
/build/x86
```

To the .gitignore file. Or better yet: Change x86dir and x64dir to be in the intermediate directory, and TargetDir to be in the target directory, and make sure the script works with that (i.e. do 'mkdir -p' to insure the target directory exists). If you do this, then no .gitignore changes are needed, as those directories are already ignored.